SCRUM-18 Öppettider, ingen ändring i css

SamuelHogberg opened the pull request.

SamuelHogberg committed:
Öppettider, ingen ändring i css

karinEkenberg reviewed (approved):
snyggt!

rfhager reviewed (changes requested):
Tänkte bara att när inga ändringar är gjorda i CSS att det är onödigt att ladda upp en fil på nytt. Kan man ta bort CSS filen från denna commiten och uppdatera länken från HTML filen till den redan befintliga CSS filen som ligger i css/style.css?
I övrigt bra jobbat! 👍

SamuelHogberg committed:
Deleted style.css

Tog bort style.css då den laddades upp i onödan. Kvar ligger oppettider.html som det ska vara.

SamuelHogberg commented:
Sådär, tog bort style.css och kvar ligger endast oppettider.html :)
